pages/pages-root-folder/software.md

```diff
@@ -29,7 +29,7 @@ This is aimed at end-users looking for a suitable tool to process their data. Th
 3. Benoit Bourassa Moreau
 4. Ingomar Gutmann
 5. P.T. (TBC)
```

Comment:
@plaresmedima I suggest we either put the full name here, or remove "P.T." and just say "TBC".

We should also consider adding affiliations of all participants, to add a bit more weight to those lists.